### PR TITLE
Update the GraphQL CLI help text

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the GraphQL CLI help text.

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help [hunk 1]
```diff
@@ -1,7 +1,5 @@
 NAME
-       ballerina-graphql - Generate the Ballerina client sources for a GraphQL config file, 
-       generate the GraphQL schema for a Ballerina GraphQL service,
-       and generate the Ballerina service sources for a GraphQL schema.
+       ballerina-graphql - Generate the Ballerina client sources for a GraphQL config file, the GraphQL schema for a GraphQL service, and vice versa.
```
Comment (ThisaruGuruge) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:2]: "and vice versa" is ambiguous here. Better to clearly state "generate GraphQL service from a schema".

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help [hunk 2]
```diff
@@ -40,9 +40,7 @@ COMMANDS
         format          Format Ballerina source files
         grpc            Generate the Ballerina sources for a given Protocol
                         Buffer definition
-        graphql         Generate the Ballerina client sources for a GraphQL config file,
-                        generate the GraphQL schema for a Ballerina GraphQL service, and 
-                        generate the Ballerina GraphQL service for a GraphQL schema
+        graphql         Generate the Ballerina client sources for a GraphQL config file, the GraphQL schema for a GraphQL service, and vice versa.
```
Comment (ThisaruGuruge) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help:43]: Ditto

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help (hunk 1 above)
Comment (praneesha) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:2]: Won't it be too long for the short description as we include a longer description also?

Comment (ThisaruGuruge) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help:2]: But shouldn't we prioritise the clarity here? We have three functionalities in the GraphQL tool. We are mentioning two, and fill it with vice-versa which I find ambiguous.

Comment (praneesha) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help]: Updated.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**